OJP/OJP_Common.xsd

```diff
@@ -420,4 +420,41 @@
 			<xs:group ref="WebLinkGroup" minOccurs="0"/>
 		</xs:sequence>
 	</xs:complexType>
+	<xs:annotation>
+		<xs:documentation>========================================== Sustainability ==========================================</xs:documentation>
+	</xs:annotation>
+	<xs:group name="SustainabilityGroup">
+		<xs:annotation>
+			<xs:documentation>Elements about sustainability. Primarily for TRIPs and LEGs.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="EmissionCO2" minOccurs="0">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="KilogramPerPersonKm">
+							<xs:annotation>
+								<xs:documentation>Kilogram by person and by kilometer of CO2 emmisions.</xs:documentation>
+							</xs:annotation>
+							<xs:simpleType>
+								<xs:restriction base="xs:float">
+									<xs:minInclusive value="0"/>
+								</xs:restriction>
+							</xs:simpleType>
+						</xs:element>
+						<xs:element name="ConfidenceLevel" minOccurs="0">
+							<xs:annotation>
+								<xs:documentation>Confidence level of the emission value in percent.</xs:documentation>
+							</xs:annotation>
+							<xs:simpleType>
```

Comment:
I think that a percentage type was recently introduced ..... it may be used here

Reply:
done